services/micro-templating-service.ts

```diff
@@ -0,0 +1,38 @@
+///<reference path="../../.d.ts"/>
+"use strict";
+
+import util = require("util");
+
+export class MicroTemplateService implements IMicroTemplateService{
+	private localVariables: IDictionary<any>;
+	private dynamicCallRegex: RegExp;
+
+	constructor(private $dynamicHelpService: IDynamicHelpService,
+		private $injector: IInjector) {
+		this.initializeLocalVariables().wait();
```

Comment:
Do you need to initialize local variables in constructor? Can we do it in `parseContent` function?